tests/ui/test_non_admin_user.py

```diff
@@ -51,3 +59,21 @@ def test_create_storageclass_rbd(self, user_factory, login_factory):
         """Login using created user"""
         obc_ui_obj = ObcUi(login_factory(user[0], user[1]))
         obc_ui_obj.check_obc_option()
+
+
+class TestUnprivilegedUserODFAccess(E2ETest):
+    """
+    Test if unprivileged user can see ODF dashboard
+    """
+
+    @ui
+    @tier2
```

Comment:
For adding it to tier2, let's seek clarity from Bipin Kunal or Harish

Reply:
Since this is a security concern, IMO, better to add it in tier1 so we detect such loopholes much early